.gitlab-ci.yml

```
@@ -1055,20 +1055,20 @@ library:ci-mathcomp_word:
   - library:ci-mathcomp
   stage: build-2
 
-library:ci-jasmin:
-  extends: .ci-template-flambda
-  needs:
-  - build:edge+flambda
-  - library:ci-stdlib+flambda
-  - plugin:ci-elpi_hb
-  - library:ci-mathcomp
-  - library:ci-mathcomp_word
-  - library:ci-mczify
-  - library:ci-algebra_tactics
-  - library:ci-ext_lib
-  - library:ci-paco
-  - library:ci-itree
-  stage: build-3+
+# library:ci-jasmin:
+#   extends: .ci-template-flambda
+#   needs:
+#   - build:edge+flambda
+#   - library:ci-stdlib+flambda
+#   - plugin:ci-elpi_hb
+#   - library:ci-mathcomp
+#   - library:ci-mathcomp_word
+#   - library:ci-mczify
+#   - library:ci-algebra_tactics
+#   - library:ci-ext_lib
+#   - library:ci-paco
+#   - library:ci-itree
+#   stage: build-3+
```

```suggestion
.library:ci-jasmin: # disabled until repaired
  extends: .ci-template-flambda
  needs:
  - build:edge+flambda
  - library:ci-stdlib+flambda
  - plugin:ci-elpi_hb
  - library:ci-mathcomp
  - library:ci-mathcomp_word
  - library:ci-mczify
  - library:ci-algebra_tactics
  - library:ci-ext_lib
  - library:ci-paco
  - library:ci-itree
  stage: build-3+
```
may be a nicer way to turn it off? keep the current version if you prefer it